lib/gates_qsim.h

```diff
@@ -230,6 +269,10 @@ struct GateRXY {
   }
 };
 
+/**
+ * A pi / 2 rotation around the X + Y axis.
+ * Analogous to the Hadamard gate, which rotates around the X + Z axis.
```

Comment:
I am not sure if this is analogous to the Hadamard gate, which rotates around the X + Z axis. This gate is a pi / 2 rotation around the X + Y axis with a global phase. The matrix is e^{i \pi / 4} e^{-i (X + Y) / \sqrt{2} \pi / 4}.

Reply:
Removed the reference to Hadamard gate.